client/app/_services/testsruns.service.js

```diff
@@ -129,6 +130,15 @@
             return isSearchActive() && !_searchParams.hasOwnProperty('query');
         }
 
+        function isModalSearchActive() {
+            const allSearchActive = isSearchActive();
+            const querySearchActive = _searchParams.hasOwnProperty('query');
+            const minimalLengthForSearch = Object.keys(DEFAULT_SC).length + 1;
```

Comment:
This doesn't work because there is can be one more additional field in `_searchParams`: `projectNames` if selected any project in projects menu